README.md

```diff
@@ -85,6 +85,7 @@ The current version of the snippet is version 7, the version is identified by th
     // name: "appInsights", // Global SDK Instance name defaults to "appInsights" when not supplied
     // ld: 0, // Defines the load delay (in ms) before attempting to load the sdk. -1 = block page load and add to head. (default) = 0ms load after timeout,
     // useXhr: 1, // Use XHR instead of fetch to report failures (if available),
+    // disableSdkLoadError: true, // Prevent the SDK from reporting load failure,
```

Comment:
Can we make this smaller, as the snippet needs to be able to be as small as possible.

something like `dle`, `dlf`, `dler` etc -- as long as the documentation has a good description this should be as small as possible (but not just `d` for example) as most consumers wont' use this as we don't want to have losts of additional code thats rarely used.